include/alpaka/math/atan/AtanUniformCudaHipBuiltIn.hpp

```diff
@@ -33,16 +33,16 @@ namespace alpaka
             template<typename TArg>
             struct Atan<AtanUniformCudaHipBuiltIn, TArg, std::enable_if_t<std::is_floating_point_v<TArg>>>
             {
-                __device__ auto operator()(AtanUniformCudaHipBuiltIn const& atan_ctx, TArg const& arg)
+                __device__ auto operator()(AtanUniformCudaHipBuiltIn const& /* atan_ctx */, TArg const& arg)
```

Comment:
Why is the name of the parameter even important here? Why not just drop it?

```suggestion
                __device__ auto operator()(AtanUniformCudaHipBuiltIn const&, TArg const& arg)
```

Reply:
This has been requested by @psychocoderhpc for documentation purposes.